doc/examples/test-examples.js

```diff
@@ -0,0 +1,36 @@
+const { readdirSync, statSync } = require('fs');
+const { join } = require('path');
+const { spawn } = require('child_process');
+const exampleDirs = readdirSync(__dirname)
+	.map(dir => join(__dirname, dir))
+	.filter(dir => statSync(dir).isDirectory());
+
+async function runner(dir) {
+	let child = spawn('npm install && npm test', {
```

Comment:
IIRC `child_process.spawn` has some gnarly Windows issues. Maybe use [`execa`](https://github.com/sindresorhus/execa) instead? It'd also give us the benefit of removing the below Promise wrapper.

Reply:
I made sure there wasn't any windows issues, but it looks like we're using `execa` elsewhere so I can switch to that.